example.env.local

```diff
@@ -1 +1,2 @@
-DB_URL=mongodb://localhost:27017/
\ No newline at end of file
+DB_URL=mongodb://localhost:27017/
+DB_NAME=local
```

Comment:
We should hard-code this instead of introducing more configurability.

Reply:
Removed

Reply:
My bad-- I meant to highlight just the DB name. The DB URL will very much change for both configurability and security. This repo is open so hardcoding the DB url would be bad :grin: